src/sinks/splunk_hec/logs/config.rs

```diff
@@ -288,6 +280,67 @@ impl HecLogsSinkConfig {
     }
 }
 
+impl ValidatableComponent for HecLogsSinkConfig {
+    fn validation_configuration() -> ValidationConfiguration {
+        let endpoint = "http://127.0.0.1:9001".to_string();
```

Comment:
Yeah that's a good point. I think it's a good idea. I'll open a separate PR for that. I'll drop it in the framework as a utility function but since it'll be isolated it should be easy to review, I'll tag you.